composer/yamls/models/bert-base.yaml

```diff
@@ -0,0 +1,84 @@
+train_dataset:
+  lm:
+    split: train
+    datadir:
+      - /mnt/gcp/datasets/wikipedia_saved_bert_128
+      - /mnt/gcp/datasets/bookcorpus_saved_bert_128
+    tokenizer_name: bert-base-uncased
+    seed: 19
+    shuffle: false
+    drop_last: false 
```

Comment:
Hmm but there is an optimization issue with `drop_last` because the very last gradient step will be out-of-proportion. Consider if the usual batch size is 2048, but the very last incomplete batch is only like 12 or so. Then the last gradient will be many times larger than usual (becaus |g| goes down as BS goes up), and so the very last update could be much bigger than we expect. Even with an optimizer like Adam, this could still produce problems. Feels bad if the last step of training is potentially harmful :( 

For multi-epoch runs with `drop_last`, as we do in ImageNet, you still have shuffling each epoch, so in expectation all samples do get seen.